discovery-provider/src/queries/response_name_constants.py

```diff
@@ -11,6 +11,10 @@
 followee_reposts = "followee_reposts"
 # array - followees of current user that have saved given track/playlist
 followee_saves = "followee_saves"
+# boolean - has current user unlocked track if premium
+has_current_user_unlocked = "has_current_user_unlocked"
```

Comment:
the signature should be enough, but in case the user has NOT unlocked, the boolean prop would be nice. the client can handle that piece though so i can remove `has_current_user_unlocked`